.circleci/config.yml

```diff
@@ -49,6 +49,21 @@ jobs:
           paths:
             - dist
 
+  e2e:
+    <<: *node_executor
+    steps:
+      - ci/pre-setup
+      - node/npm-install
+      - run:
+          name: "Build checkout-js"
+          command: npm run build
+      - run:
+          name: "Install playwright"
+          command: npx playwright install --with-deps
+      - run:
+          name: "Run e2e tests"
+          command: npx playwright test
```

Comment:
After this step, could consider uploading the artifacts to CircleCI (screenshots for failed tests etc...) - see https://circleci.com/docs/artifacts#uploading-artifacts

Reply:
Hi @davidchin , CircleCI is configured as suggested. 

Here is a demo for e2e job failure scenario: job [link](https://app.circleci.com/pipelines/github/bigcommerce/checkout-js/5048/workflows/9b97619d-ad00-411a-bb77-4f18e2473996/jobs/12965), artifacts [link](https://app.circleci.com/pipelines/github/bigcommerce/checkout-js/5048/workflows/9b97619d-ad00-411a-bb77-4f18e2473996/jobs/12965/artifacts), HTML test report [viewable link](https://output.circle-artifacts.com/output/job/1e155c0d-6389-4014-8ace-8cbcac68afa5/artifacts/0/artifact-file/index.html).

I've also configured `playwright` to generate comprehansive trace-recordings([doc](https://playwright.dev/docs/trace-viewer)) for failed tests, which can be found on the artifact page.